tests/test_tti.py

```diff
@@ -73,7 +73,8 @@ def ricker_source(t, f0):
                                   time_order=2, space_order=space_order)
 
     solver_tti = AnisotropicWaveSolver(model, source=source, receiver=receiver,
-                                       time_order=2, space_order=space_order)
+                                       time_order=2, space_order=space_order,
+                                       dse='basic')
```

Comment:
I guess we won't need to have ``dse`` here as soon as the advanced mode bug is fixed